src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs

```diff
@@ -18,18 +18,22 @@ public static partial class ImageSourceExtensions
 			var glyph = (NSString)imageSource.Glyph;
 
 			var attString = new NSAttributedString(glyph, font, color);
-			var imagesize = glyph.GetSizeUsingAttributes(attString.GetUIKitAttributes(0, out _));
+			var attributes = attString.GetUIKitAttributes(0, out _);
+			if (attributes != null)
```

Comment:
@mattleibow  any idea a better way here if this is null to get the imagesize  var below ?